app/models/host_aggregate.rb

```diff
@@ -0,0 +1,43 @@
+class HostAggregate < ApplicationRecord
+  include NewWithTypeStiMixin
+  include Metric::CiMixin
+  include EventMixin
+  include ProviderObjectMixin
+  include VirtualTotalMixin
+
+  acts_as_miq_taggable
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+
+  # FIXME: I don't think this is needed here
```

Comment:
@sseago, I think it is more complex:
- a host aggregates can span across availability zone  (AZ) 
- a host can be in multiple aggregates

https://kimizhang.wordpress.com/2013/08/26/openstack-zoning-regionavailability-zonehost-aggregate/

And there is also the possiblity to Create a host aggregate that is exposed to users as an availability zone but it is not in an Availability Zone 

https://blog.russellbryant.net/2013/05/21/availability-zones-and-host-aggregates-in-openstack-compute-nova/

So, We need the relationships between AZ and Host Aggregates, and relationships between Host Aggregates and Hosts.

After we do have discovery working the plan will be:
From User point of View, during provisioning user cans Select AZ and then Host Aggregate  in this AZ except if the AZ is an host aggregate itself.
From Admin point of view, admin will be able to manage Host Aggregate, create a new one, add/remove host, change AZ etc...